app/src/main/java/com/example/blockchain/di/AppModule.kt

```diff
@@ -10,12 +11,19 @@ import dagger.Provides
 import javax.inject.Singleton
 
 @Module
-class AppModule(private val context: Context) {
+object AppModule {
+
+    @JvmStatic
```

Comment:
@adrianaisemberg - JvmStatic and Singleton both not needed right?

Reply:
JvmStatic **is** needed (because now AppModule is an object (which doesn't have any dependencies therefore Dagger binds it automatically for you).
However - Singleton - that's up to your design.
Begin singleton means it will only be called once per component, not once per app runtime.